graphene_django/range_types.py

```diff
@@ -0,0 +1,29 @@
+from graphene import ObjectType
+from graphene import (
+    Float,
+    Int,
+    DateTime,
+    Date
+)
+
+class RangeResolver:
+    def resolve_lower(parent, info):
+        return parent.lower
+    def resolve_upper(parent, info):
+        return parent.upper
```

Comment:
If the resolvers just pick attributes on the parent object that are the same name as the field then you don't need these resolvers.

Reply:
I'll try